prac_06/guitar.py

```diff
@@ -0,0 +1,30 @@
+"""cp1404 practical of guitar class"""
+
+
+class Guitar:
+    def __init__(self, name="", year=0, cost=0):
+        self.name = name
+        self.year = year
+        self.cost = cost
+
+    def __str__(self):
+        return "{} ({}) : {}".format(self.name, self.year, self.cost)
+
+    def get_age(self):
+        return 2021 - self.year
+
+    def is_vintage(self):
+        if 2021 - self.year > 50:
```

Comment:
Do you see how you use these "magic numbers"?
Consider: https://github.com/CP1404/Starter/wiki/Programming-Patterns#constants